PowerEditor/src/MISC/Common/Common.h

```diff
@@ -83,12 +82,11 @@ const bool dirDown = false;
 	#define generic_strtok strtok
 	#define generic_strftime strftime
 	#define generic_fprintf fprintf
-	#define generic_sprintf sprintf
+	#define generic_sprintf snprintf
 	#define generic_sscanf sscanf
 	#define generic_fopen fopen
 	#define generic_fgets fgets
 	#define generic_stat _stat
-	#define generic_sprintf sprintf
 	#define COPYDATA_FILENAMES COPYDATA_FILENAMESA
 #endif

```

Comment:
Hang on a second..... You're getting rid of the `wchar_t`-accepting version? What? How does this even compile??!?